src/dosbox.cpp

```diff
@@ -45,8 +45,9 @@
 #include "video.h"
 
 bool shutdown_requested = false;
-MachineType machine;
-SVGACards svgaCard;
+
+MachineType machine   = MachineType::Invalid;
```

Comment:
How about `MachineType::None` for consistency? Better yet, get rid of it in a separate commit (the init code needs it to be "none" for a while, so it might be tricky).

Reply:
@Torinde you're getting into scope creep territory again. Read again what our mission goal is (hint: games).

This will not happen.

Reply:
Changed `Init` to `None`.

Getting rid of it (if we don't want to use `std::optional` or any separate variable) indeed might be tricky, as `shader_manager.cpp` uses this value and I'm not sure what the init flow looks like (and which use cases are possible) - I'm probably not the right person for this job :)

Reply:
Thanks, I might do it later when I rework the shader loading in the coming months.